adafruit_platformdetect/constants/boards.py

```diff
@@ -408,6 +408,7 @@
             "nvidia,p3768-0000+p3767-0004",
             "nvidia,p3509-0000+p3767-0005",
             "nvidia,p3768-0000+p3767-0005",
+            "nvidia,p3768-0000+p3767-0005-super",
```

Comment:
Thanks for this!

Also add this one please: "nvidia,p3768-0000+p3767-0005-supernvidia"